Cargo.toml

```diff
@@ -9,6 +9,9 @@ repository = "https://github.com/dusk-network/nstack"
 keywords = ["merkle", "datastructure", "stack"]
 
 [dependencies]
-microkelvin = "0.7"
+microkelvin = "0.9.0-rc.0"
```

Comment:
Remove the `.0`, it should be just `0.9.0-rc`.
In that way cargo is able to automatically update to any possible `rc.1`, `rc.2`, etc. Also, it will be able to use directly `0.9.0` once released.